src/confdb/confdb.h

```diff
@@ -224,6 +224,7 @@
 /* Secrets Service */
 #define CONFDB_SEC_CONF_ENTRY "config/secrets"
 #define CONFDB_SEC_CONTAINERS_NEST_LEVEL "containers_nest_level"
+#define CONFDB_SEC_MAX_ENTRIES "max_entries"
```

Comment:
I would prefer if the option was named max_secrets or using some other name that would explicitly make it clear this is about secrets. The reason is that soon we will store Kerberos credential caches as well in the secrets responder, but I don't think the setting to limit the number of secrets should limit the number of ccaches. And max_entries to me sounds like it would put a cap on both, globally..